.prettierrc.yml

```diff
@@ -0,0 +1,19 @@
+tabWidth: 2
```

Comment:
It will be better to use the same config that we have in frontend team for js

https://github.com/VilnaCRM-Org/website/blob/main/.prettierrc